app/resources/base.html.template

```diff
@@ -21,6 +21,15 @@
 
 <head>
   {% block head %}
+
+    {# If there is no Javascript support, redirect to the light URL. #}
+    {% if env.ui != "light" %}
+      <noscript>
+        <style>html{display:none;}</style>
```

Comment:
Does this not hide the display of the "click here" message too? You might have to apply `display: none` to a wrapper class, and put the fallback message outside that wrapper.

Reply:
Good catch. Done.